lib/helper/Protractor.js

```diff
@@ -227,36 +228,36 @@ class Protractor extends SeleniumWebdriver {
   /**
    * {{> ../webapi/waitForElement }}
    */
-  waitForElement(locator, sec = null) {
-    sec = sec || this.options.waitForTimeout;
-    let el = element(guessLocator(locator) || by.css(locator));
+  waitForElement(locator, aSec = null) {
```

Comment:
Why do you prefer `aSec` instead of `sec`?

Reply:
This showed up thanks to the TS compiler warnings for JS files ! :joy: 

If the parameter is null and waitForTimeout is undefined, null will get multiplied by 1000 which can cause super funky bugs.

Reply:
Can you revert name of param back to `sec` because it is already documented as `sec`.
You can use `aSec` as an internal variable name )
Thanks!